css/services.css

```diff
@@ -0,0 +1,362 @@
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100%;
+  font-family: "Titillium Web", sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: "Bangers", cursive;
+  letter-spacing: 1px;
+  margin-bottom: 15px;
+}
+
+button:hover {
+  -webkit-transform: scale(1.1);
+  -ms-transform: scale(1.1);
+  transform: scale(1.1);
+  transition: 0.05s;
+}
+
+.container {
+  
+  width: 1024px;
+  margin: 0 auto;
+  padding: 40px;
+}
+
+/* div {
+  margin: 10px 30px;
+} */
+
+@media (max-width: 800px) {
+  .container {
+    width: 100%;
+  }
+  .top {
+    display: flex;
+    flex-direction: column-reverse;
+    align-items: center;
+  }
+
+  .top nav {
+    margin: 20px auto;
+    width: 100%;
+  }
+
+  .services-header-img {
+    display: none;
+  }
+
+  .head {
+    text-align: center;
+  }
+
+  h1 {
+    margin-bottom: 30px;
+  }
+}
+
+@media (max-width: 500px) {
+  .top {
+    display: flex;
+    flex-direction: column-reverse;
+    align-items: center;
+  }
+
+  nav {
+    margin: 30px 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  nav a {
+    border: 1px solid gray;
+    border-left: 0;
+    border-right: 0;
+    padding: 5px 0;
+    width: 100%;
+    text-align: center;
+    text-decoration: none;
+  }
+
+  .first {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+}
+.top {
+  margin: 20px 0;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.top nav {
+  width: 70%;
+  display: flex;
+  justify-content: space-between;
+}
+
+nav a {
+  color: gray;
+  text-decoration: none;
+}
+
+nav a:hover {
+  color: lightblue;
+}
+
+.services-header-img {
+  width: 100%;
+  margin-bottom: 30px;
+}
+
+.head {
+  display: flex;
+  flex-direction: column;
+  border-bottom: 2px solid black;
+  padding-bottom: 30px;
+}
+
+h1 {
+  font-size: 25px;
+}
+
+h2,
+h3,
+h4,
+h5 {
+  font-size: 15px;
+}
+
+.first div {
+  
+  margin-bottom: 30px;
+  padding: 20px;
+}
+
+.first {
+  display: flex;
+  align-content: flex-start;
+  justify-content: space-between;
+  padding-top: 30px;
+}
+
+.gr {
+  width: 48%;
+  background: lightgray;
+  border: 1px solid black;
+}
+
+@media (max-width: 500px) {
+  .gr {
+    width: 100%;
+  }
+}
+
+button {
+  width: 150px;
+  margin-top: 20px;
+}
+
+@media(max-width: 500px) {
+  button{
+    width: 30%;
+  }
+}
+.dots {
+  border-top: 2px solid black;
+  width: 100%;
+  margin: 30px 0;
+  padding-top: 30px;
+  display: flex;
+  justify-content: space-between;
+}
+
+@media(max-width: 500px) {
+  .dots {
+    display: flex;
+    flex-direction: column-reverse;
+    align-items: center;
+  }
+}
+
+ul {
+  width: 200px;
+  margin-left: 20px;
+  margin-top: 20px;
+}
+
+li {
+  margin: 20px 0;
+  box-sizing: border-box;
+}
+
+.bottomright {
+  width: 45%;
+  font-size: 15px;
+}
+
```

Comment:
I like how you placed media queries in-line. This is a slightly more advanced technique that most people couldn't get to work, and this will help you adjust more easily when we get to pre-processing as this is how it's done.